addons/mail/static/tests/m2x_avatar_user_tests.js

```diff
@@ -58,189 +58,57 @@ QUnit.module('mail', {}, function () {
     });
 
     QUnit.test('many2one_avatar_user widget in list view', async function (assert) {
-        assert.expect(5);
+        assert.expect(2);
 
         const { widget: list } = await start({
+            hasChatWindow: true,
             hasView: true,
             View: ListView,
             model: 'foo',
             data: this.data,
             arch: '<tree><field name="user_id" widget="many2one_avatar_user"/></tree>',
-            mockRPC(route, args) {
-                if (args.method === 'read') {
-                    assert.step(`read ${args.model} ${args.args[0]}`);
-                }
-                return this._super(...arguments);
-            },
         });
 
-        mock.intercept(list, 'open_record', () => {
-            assert.step('open record');
-        });
-
-        assert.strictEqual(list.$('.o_data_cell span').text(), 'MarioLuigiMarioYoshi');
-
-        // sanity check: later on, we'll check that clicking on the avatar doesn't open the record
-        await dom.click(list.$('.o_data_row:first span'));
-
         await dom.click(list.$('.o_data_cell:nth(0) .o_m2o_avatar > img'));
-        await dom.click(list.$('.o_data_cell:nth(1) .o_m2o_avatar > img'));
-        await dom.click(list.$('.o_data_cell:nth(2) .o_m2o_avatar > img'));
+        assert.containsOnce(document.body, '.o_ChatWindow', 'Chat window should be opened');
 
```

Comment:
Can remove this LOC (successive assertions should not be separated by newlines).

Just make a newline when it's another step of test, i.e. another group of user interactions followed by assertion(s). The shape helps get the complexity of test at a quick glance. And also the newline should let you know that you could split the test there.

Reply:
The comment is on empty line 89/74. Github shows more context above.